nece/managers.py

```diff
@@ -96,8 +96,11 @@ def language(self, language_code):
 
     def _clone(self):
         """Override `_clone` method in order to inject the `language_code`."""
+        language_code = self._language_code
```

Comment:
We can even more simplify this:
`language_code = self._language_code or get_language().replace("-", "_")`